trace-demo/README.md

```diff
@@ -1,176 +1,3 @@
-   ```
-   ```bash
-   open http://localhost:8089
-   ```
-
-## Advanced
-
-### Browser Instrumentation and missing root spans
-
-The frontend application of the demo uses [browser instrumentation](https://opentelemetry.io/docs/demo/services/frontend/#browser-instrumentation). Because of that, the root span of a trace is created externally to the cluster and will not be captured with the described setup. In Jaeger, you can see a warning at the first span indicating that there is a parent span that is not being captured.
-
-In order to capture the spans reported by the browser, you need to expose the trace endpoint of the collector and configure the frontend to report the spans to that exposed endpoint.
-
-To expose the frontend web app and the relevant trace endpoint, define an Istio Virtualservice as in the following example.
->**CAUTION** The example shows an insecure way of exposing the trace endpoint. Do not use it in production.
-```bash
-export CLUSTER_DOMAIN={my-domain}
-
-cat <<EOF | kubectl -n $K8S_NAMESPACE apply -f -
-apiVersion: networking.istio.io/v1beta1
-kind: VirtualService
-metadata:
-  name: frontend
-spec:
-  gateways:
-  - kyma-system/kyma-gateway
-  hosts:
-  - frontend.$CLUSTER_DOMAIN
-  http:
-  - route:
-    - destination:
-        host: telemetry-otlp-traces.kyma-system.svc.cluster.local
-        port:
-          number: 4318
-    match:
-    - uri:
-        prefix: "/v1/traces"
-    corsPolicy:
-      allowOrigins:
-      - prefix: https://frontend.$CLUSTER_DOMAIN
-      allowHeaders:
-      - Content-Type
-      allowMethods:
-      - POST
-      allowCredentials: true
-  - route:
-    - destination:
-        host: $HELM_OTEL_RELEASE-frontend.$K8S_NAMESPACE.svc.cluster.local
-        port:
-          number: 8080
-EOF
-```
-
-Then, update the frontend configuration. To do that, run the following helm command and set the additional configuration:
-```bash
-helm upgrade --version 0.22.2 --install --create-namespace -n $K8S_NAMESPACE $HELM_OTEL_RELEASE open-telemetry/opentelemetry-demo \
--f https://raw.githubusercontent.com/kyma-project/examples/main/trace-demo/values.yaml \
---set 'components.frontend.envOverrides[1].name=PUBLIC_OTEL_EXPORTER_OTLP_TRACES_ENDPOINT' \
---set "components.frontend.envOverrides[1].value=https://frontend.$CLUSTER_DOMAIN/v1/traces"
-```
-
-Afterwards, the web application will be accessible in your browser at `https://frontend.$CLUSTER_DOMAIN`. Using the developer tools of the browser you should see that requests to the traces endpoint are successful. Now, every trace should have a proper root span recorded.
-## Cleanup
-
-When you're done, you can remove the example and all its resources from the cluster by calling Helm:
-
-```bash
-helm delete -n $K8S_NAMESPACE $HELM_OTEL_RELEASE
-```
+The document has been moved to [Telemetry Integration Guides](https://github.com/kyma-project/telemetry-manager/tree/main/docs/user/integration).
```

Comment:
I prefer the homepage link, what do you think?

Reply:
I thought we should link to the integration guides, that's why I added that link, but you can try to convince me ;)
Pro the integration guides: The already-annoyed reader doesn't have to search further.

Reply:
I mean that kind of link https://kyma-project.io/#/telemetry-manager/user/integration/loki/README so that it is nicely rendered

Reply:
Here we go: https://github.com/kyma-project/examples/pull/278/commits/8521c133cce83e4df77a0d2e0740b62a3b4a2fab